src/index.tsx

```diff
@@ -125,8 +131,37 @@ async function activatePlugin(
 ): Promise<void> {
   const { commands } = app;
   const trans = translator.load('jupyterlab');
```

Comment:
You can move these 2 lines (126-127) down to after the server extension check block.

Reply:
`trans` is used in the `showDialog` call, but I can move the `commands` instantiation lower.